tests/test_dfrepos.py

```diff
@@ -0,0 +1,123 @@
+import io
+import unittest.mock
+import json
+
+from dffml.df.types import DataFlow, Input
+from dffml.df.memory import MemoryOrchestrator
+from dffml.operation.dfrepos import (
+    run_dataflow_on_repo,
+    RunDataFlowOnRepoConfig,
+)
+from dffml.operation.output import GetSingle
+from tests.integration.common import IntegrationCLITestCase, relative_chdir
+from dffml.service.dev import Develop
+
+from examples.shouldi.shouldi.safety import safety_check
+from examples.shouldi.shouldi.bandit import run_bandit
+from dffml.df.base import OperationImplementationContext
+
+
+class TestRunDataFlowOnRepo(IntegrationCLITestCase):
+    async def test_run(self):
+        self.required_plugins("shouldi")
+        stdout = io.StringIO()
+
+        with relative_chdir("..", "..", "examples", "shouldi"):
+            with unittest.mock.patch("sys.stdout.buffer.write") as write:
+                await Develop.cli("export", "shouldi.cli:DATAFLOW")
+            shouldi_dataflow = DataFlow._fromdict(
+                **json.loads(write.call_args[0][0])
+            )
```

Comment:
Using the `shouldi` dataflow is a great test. However I think we probably want to avoid it here since it does network requests and slows down our already long tests for the main repo. This will also avoid the event that the number of issues might change.

It would be good to take the test dataflow in `tests/test_df.py` and make it a global variable. Then you can import it easily from this file and use that as the test dataflow.

https://github.com/intel/dffml/blob/85cd82379042d543fb280163fee68b0043cc94da/tests/test_df.py#L181-L184